godot-project/scenes/prefabs/Player/PlayerCamera.gd

```diff
@@ -0,0 +1,7 @@
+extends Camera2D
+
+
+
+func _process(delta):
+	offset_h = lerp(-0.5, 0.5, get_viewport().get_mouse_position().x / get_viewport_rect().size.x)
+	offset_v = lerp(-0.5, 0.5, get_viewport().get_mouse_position().y / get_viewport_rect().size.y)
```

Comment:
It's a good idea that the PlayerCamera logic is separate from the Player, but this is also inconsistent and more importantly the camera is still movable during IMV cutscenes, which shouldn't be the case.
Either move the logic here to the player, or replicate and move the logic from player here.